src/Servers/Kestrel/Core/src/Internal/Http/Http1OutputProducer.cs

```diff
@@ -117,19 +222,30 @@ public void WriteResponseHeaders(int statusCode, string reasonPhrase, HttpRespon
                 writer.Commit();
 
                 _unflushedBytes += writer.BytesCommitted;
+                _autoChunk = autoChunk;
             }
         }
 
         public void Dispose()
```

Comment:
Make sure this is still being called.

Reply:
It is called here https://github.com/aspnet/AspNetCore/blob/7d52a0c9e3da57839e227f814ea73d31351578f8/src/Servers/Kestrel/Core/src/Internal/Http/Http1Connection.cs#L79